layouts/partials/jslibraries.html

```diff
@@ -21,6 +21,150 @@
 <!-- Pinggy Tunnel Shortcode -->
 <script src="/js/pinggytunnelshortcode.js"></script>
 
+<script>
+  document.addEventListener("alpine:init", () => {
+    Alpine.store("advModal", {
+      httpConfig: {
+        localPort: 8000,
```

Comment:
These codes are relevant to index only. Should have gone to index.js isn't it?

Reply:
Have changed that!